.gitignore

```diff
@@ -18,3 +18,4 @@ website/node_modules
 *.bak
 *~
 .*.swp
+.idea/
```

Comment:
@hngkr so that this doesn't end up in the final merged history, could you squash your fixup into the original commit?

You can do this as follows, while on your branch:

* Run ``git rebase -i master``
* When your editor pops up to edit the history, change "pick" in front of that commit to "fixup", then save the changes and exit the editor.
* Git should then combine those two commits together, using the commit message from your previous commit.
* You can then update the remote branch using ``git push --force``, so the github server will let you change the history.